### ISSUE TITLE
Auto inherit a configuration in 1_Configure.py when the config exists.

### ISSUE THREAD

**Issue by Knitschi** [Knitschi/CPFBuildscripts#7]
Currently we need to specify a configuration twice if we want to create a local one with an existing name.

```
1_Configure.py Gcc-shared-debug --inherits Gcc-shared-debug
```

This is tedious and quite a common case. Using only

```
1_Configure.py Gcc-shared-debug
```

should do the same thing.

**Comment by Knitschi** [comment]
This should be extended even further.

`2_Generate.py` should execute `1_Configure.py` if the configuration file has not yet been created.
`3_Make.py` should execute `2_Generate.py` if the cache file has not yet been created.

**Comment by Knitschi** [comment]
Fixed with d4507f5be800c15abed5c13215712804800d97a6
